Updates

Abili opened the pull request.

Abili committed:
update payment method

Abili committed:
update payment method

Abili committed:
Update index.html

Abili committed:
payment

Abili committed:
payment

Abili committed:
Merge branch 'main' into updates

happy-janet commented:
https://meet.google.com/nzv-srrb-yue

On Sat, Sep 7, 2024 at 12:48 PM Abili Isaac ***@***.***>
wrote:

> @Abili <https://github.com/Abili> requested your review on: #4
> <https://github.com/happy-janet/Raha-Voyage-Safaris/pull/4> Updates.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/happy-janet/Raha-Voyage-Safaris/pull/4#event-14168988379>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BGJTZLZ4BFZUF2Z2AOCLUM3ZVLDVBAVCNFSM6AAAAABNZ36NM6VHI2DSMVQWIX3LMV45UABCJFZXG5LFIV3GK3TUJZXXI2LGNFRWC5DJN5XDWMJUGE3DQOJYHAZTOOI>
> .
> You are receiving this because your review was requested.Message ID:
> ***@***.***
> com>
>

Abili commented:
@import url('
***@***.***;300;400;600;700&display=swap
');

:root{
--orange:#ffa500;
}


*{
font-family: 'Nunito', sans-serif;
margin: 0px;
padding: 0px;
box-sizing: border-box;
text-transform: capitalize;
outline: none;;
border: none;
text-decoration: none;
transition:all .2s linear;
}
*::selection{
background: var(--orange);
color: white;
}
html{
font-size: 62.5%;
overflow-x: hidden;
scroll-padding-top: 6rem;
scroll-behavior: smooth;
}
section{
padding:2rem 9%;

}
.heading{
text-align: center;
padding: 2.5rem 0;
}
.heading span{
font-size: 3.5rem;
background: rgba(255, 165, 0, .2);
color: var(--orange);
border-radius: .5rem;
padding: .2rem 1rem;
}
.heading span.space{
background: none;
}
/* Header styling */
header {
display: flex;
align-items: center;
padding: 20px;
background-color: #2c3e50; /* Dark background for the header */
}
#menu-bar {
font-size: 24px;
color: #fff; /* White color for the menu icon */
margin-right: 20px;
cursor: pointer;
text-align: left;
}
.logo {
display: flex;
align-items: center;
text-decoration: none;
font-size: 24px;
color: #fff;
font-family: 'Arial', sans-serif;
}
.logo-image {
width: 50px; /* Adjust size according to your logo */
height: auto;
margin-right: 10px; /* Space between logo and text */
}
.logo span {
font-weight: bold;
color: #e74c3c; /* Color for 'Raha Voyage' */
}
.logo {
color: #fff; /* Color for 'Safaris' */
}
/* Media Query for smaller screens */
@media (max-width: 768px) {
header {
flex-direction: column;
align-items: flex-start;
}
.logo-image {
width: 40px; /* Smaller size for the logo on mobile */
}
.logo {
font-size: 20px;
}
#menu-bar {
margin-bottom: 10px;
}
}
header{
position: fixed;
top: 0; left: 0; right: 0;
background: #333;
z-index: 1000;
display: flex;
align-items: center;
justify-content: space-between;
padding: 2rem 9%;
}
header .logo{
font-size: 2.5rem;
font-weight: bolder;
color: white;
text-transform: uppercase;
}

header .logo span{
color: var(--orange);
}
header .navbar a{
color: white;
font-size: 2rem;
margin: 0 .8rem;
}
header .navbar a:hover{
color: var(--orange);
}
header.icons i{
font-size: 2.5rem;
color: white;
cursor: pointer;
margin-right: 2rem;
}
header .icons i:hover{
color: var(--orange);
}
header .search-bar-container{
position: absolute;
top: 100%; left: 0; right: 0;
padding: 1.5rem 2rem;
background: #333;
border-top: .1rem solid rgba( 255,255,255,.2);
display: flex;
align-items: center;
z-index: 1001;
clip-path: polygon(0 0,100% 0, 100% 0, 0 0);
}
header .search-bar-container.active{
clip-path: polygon(0 0,100% 0, 100% 100%, 0 100%);
}
header .search-bar-container #search-bar{
width: 100%;
padding: 1rem;
text-transform: none;
color: #333;
font-size: 1.7rem;
border-radius: .5rem;
}
header .search-bar-container label{
color: white;
cursor: pointer;
font-size: 3rem;
margin-left: 1.5rem;
}
header .search-bar-container label:hover{
color: var(--orange);
}
.btn{
display: inline-block;
margin-top: 1rem;
background: var(--orange);
color: white;
padding: .8rem 3rem;
border: .2rem solid var(--orange);
cursor: pointer;
font-size: 1.7rem;
}
.btn:hover{
background: rgba(255, 165, 0, .2);
color:var(--orange);
}
.login-form-container{
position:fixed;
top: -120%; left: 0;
z-index: 10000;
min-height: 100vh;
width: 100%;
background: rgba(0,0,0,0.7);
display:flex;
align-items: center;
justify-content: center;
}
.login-form-container.active{
top: 0;
}
.login-form-container form{
margin:2rem;
padding: 1.5rem 2rem;
border-radius: .5rem;
background: white;
width: 50rem;
}

.login-form-container form h3{
font-size: 3rem;
color: #444;
text-transform:uppercase;
text-align: center;
padding: 1rem 0;
}
.login-form-container form .box{
width:100%;
padding: 1rem;
font-size: 1.7rem;
color: #333;
margin: .6rem 0;
border: .1rem solid rgba(0,0,0,0.3);
text-decoration: none;
}
.login-form-container form .box:focus{
border-color: var(--orange);
}
.login-form-container form #remember{
margin: 2rem 0;
}
.login-form-container form label{
font-size: 1.5rem;
}
.login-form-container form .btn{
display: block;
width: 100%;
}
.login-form-container form p{
padding: .5rem 0;
font-size: 1.5rem;
color: #666;
}
.login-form-container form p a{
color: var(--orange);
}
.login-form-container form p a:hover{
color: #333;
text-decoration: underline;
}
.login-form-container #form-close{
position: absolute;
top:2rem; right: 3rem;
font-size: 5rem;
color:white;
cursor: pointer;
}
#menu-bar{
color: white;
border: .1rem solid white;
border-radius: .5rem;
font-size: 3rem;
padding: .5rem 1.2rem;
cursor: pointer;
display: none;
}

.home{
min-height: 100vh;
display: flex;
align-items: center;
justify-content: center;
flex-flow: column;
position: relative;
z-index: 0;
}
.home .content{
text-align: center;
}
.home .content h3{
font-size: 4.5rem;
color: white;
text-transform: uppercase;
text-shadow: 0 .3rem .5rem rgba(0,0,0,0.1);
}
.home .content p{
font-size: 2.5rem;
color: white;
padding: 5rem 0;
}
.home .video-container video{
position: absolute;
top: 0; left: 0;
z-index: -1;
height: 100%;
width: 100%;
object-fit: cover;
}

.home .controls{
padding: 1rem;
border-radius: 5rem;
background: rgba(0,0,0,0.7);
position: relative;
top: 10rem;
}
.home .controls .vid-btn{
height: 2rem;
width: 2rem;
display: inline-block;
border-radius: 50%;
background: white;
cursor: pointer;
margin: 0 .5rem;
}

.home .controls .vid-btn.active{
background: var(--orange);
}

.book .row{
display: flex;
flex-wrap: wrap;
gap:1.5rem;
align-items: center;
}

.book .row .image{
flex: 1 1 40rem;
}
.book .row .image img{
width: 100%;
height: 550px;
}
.book .row form{
flex: 1 1 40rem;
padding: 2rem;
box-shadow: 0 1rem 2rem rgba(0,0,0,0.1);
border-radius: .5rem;
}
.book .row form .inputBox{
padding: .5rem 0;
}
.book .row form .inputBox input{
width: 100%;
padding: 1rem;
border: 1rem solid rgba(0,0,0,0.1);
font-size: 1rem;
color: #333;
text-transform: none;
}
.book .row form .inputBox h3{
font-size: 2rem;
padding: 1rem 0;
color: #666;
}

.packages .box-container{
display: flex;
flex-wrap: wrap;
gap: 2rem;
}
.packages .box-container .box{
flex: 1 1 30rem;
border-radius: .5rem;
overflow: hidden;
box-shadow: 0 1rem 2rem rgba(0,0,0,0.1);
}
.packages .box-container .box img{
height:25rem;
width: 100%;
object-fit: cover;
}
.packages .box-container .box .content{
padding: 2rem;
}
.packages .box-container .box .content h3{
font-size: 2rem;
color: #333;
}
.packages .box-container .box .content h3 i{
color: var(--orange);
}

.packages .box-container .box .content p{
font-size: 1.7rem;
color: #333;
padding: 1rem 0;
}
.packages .box-container .box .content .stars i{
font-size: 1.7rem;
color: var(--orange);
}
.packages .box-container .box .content .price{
font-size: 2rem;
color: #333;
padding-top: 1rem;
}

.packages .box-container .box .content .price span{
color: #666;
font-size: 1.5rem;
text-decoration: line-through;
}

.services .box-container{
display: flex;
flex-wrap: wrap;
gap:1.5rem;
}
.services .box-container .box{
flex: 1 1 30rem;
border-radius: .5rem;
padding: 1rem;
text-align: center;

}

.services .box-container .box i{
padding:1rem;
font-size: 5rem;
color: var(--orange);
}
.services .box-container .box h3{
font-size: 2.5rem;
color: #333;
}

.services .box-container .box p{
padding: 1rem 0;
font-size: 1.5rem;
color: #666;
}

.services .box-container .box:hover{
box-shadow: 0 1rem 2rem rgba(0,0,0,0.1);
}

.gallery .box-container{
display: flex;
flex-wrap: wrap;
gap: 1.5rem;
}
.gallery .box-container .box{
overflow: hidden;
box-shadow: 0 1rem 2rem rgba(0,0,0,0.1);
border: 1rem solid white;
border-radius: .5rem;
flex: 1 1 30rem;
height: 25rem;
position:relative;
}

.gallery .box-container .box img{
height: 100%;
width: 100%;
object-fit: cover;
}
.gallery .box-container .box .content{
position: absolute;
top: -100%; left: 0;
height: 100%;
width: 100%;
text-align: center;
background: rgba(0,0,0,0.7);
padding: 2rem;
padding-top: 5rem;
}
.gallery .box-container .box:hover .content{
top: 0;
}
.gallery .box-container .box .content h3{
font-size: 2.5rem;
color: var(--orange);
}
.gallery .box-container .box .content p{
font-size: 1.5rem;
color: white;
padding: .5rem 0;
}

.review .box{
padding: 2rem;
text-align: center;
box-shadow: 0 1rem 2rem rgba(0,0,0,0.1);
border-radius: .5rem;
}

.review .box img{
height: 13rem;
width: 13rem;
border-radius: 50%;
object-fit: cover;
margin-bottom: 1rem;
}

.review .box h3{
color: #ffa500;
font-size: 2.5rem;
}
.review .box p{
color: #666;
font-size: 1.5rem;
padding: 1rem 0;
}
.review .box .stars i{
color: var(--orange);
font-size: 1.7rem;
}

.contact .row{
display: flex;
flex-wrap: wrap;
gap: 1.5rem;
align-items: center;
}
.contact .row .image{
flex: 1 1 35rem;
}
.contact .row .image img{
width: 100%;
height: 350px;
}
.contact .row form{
flex: 1 1 50rem;
padding: 2rem;
box-shadow: 0 1rem 2rem rgba(0,0,0,0.1);
border-radius: .5rem;
}

.contact .row form .inputBox{
display: flex;
flex-wrap: wrap;
justify-content: space-between;
}
.contact .row form ,.inputBox input, .contact .row form textarea{
width: 49%;
margin: 1rem 0;
padding: 1rem;
font-size: 1.7rem;
color: #333;
background: #f7f7f7;
text-transform: none;
}

.contact .row form textarea{
height: 15rem;
resize: none;
width: 100%;
}



.footer{
background: #333;
}

.footer .box-container{
display: flex;
flex-wrap: wrap;
gap: 1.5rem;
}
.footer .box-container .box{
padding: 1rem 0;
flex: 1 1 25rem;
}
.footer .box-container .box h3{
font-size: 2.5rem;
padding: .7rem 0;
color: white;
}
.footer .box-container .box p{
font-size: 1.5 rem;
padding: .7rem 0;
color: white;
}
.footer .box-container .box a{
display: block;
font-size: 1.5rem;
padding: .7rem 0;
color: white;
}
.footer.box-container .box a:hover{
color: var(--orange);
text-decoration: underline;
}

.footer .credit{
text-align: center;
padding: 2rem 1rem;
margin-top: 1rem;
font-size: 2rem;
font-weight: normal;
color: white;
border-top: .1rem solid rgba( 255,255,255,.2)
}
.footer .credit span{
color: var(--orange);
}






/* media queries */

@media (max-width:1200px){
html{
font-size: 55%;
}
}
@media (max-width:991px){
header{
padding: 2rem;
}
section{
padding: 2rem;
}
}
@media (max-width:768px){
#menu-bar{
display: initial;
}
header .navbar{
position: absolute;
top: 100%; right: 0; left: 0;
background: #333;
border-top: .1rem solid rgba( 255,255,255,.2);
padding: 1rem 2rem;
clip-path:polygon(0 0, 100% 0, 100% 0,0 0) ;
}
header .navbar.active{
clip-path: polygon(0 0,100% 0, 100% 100%, 0 100%);
}
header .navbar a{
display: block;
border-radius: .5rem;
padding: 1.5rem;
margin: 1.5rem 0;
background: #222;
}
}
@media (max-width:450px){
html{
font-size: 50%;
}


.heading span{
font-size: 2.5rem;
}
.contact .row form .inputBox input{
width: 100%;
}
}


/* General container styling */
#about-us {
padding: 50px 20px;
background-color: #f9f9f9; /* Light background color */
font-family: 'Arial', sans-serif;
}
#about-us .container {
max-width: 1200px;
margin: 0 auto;
}
/* Headings */
#about-us h2, #about-us h3 {
color: #ffa500; /* Darker color for headings */
text-align: center;
margin-bottom: 20px;
}
#about-us h2 {
font-size: 36px;
}
#about-us h3 {
font-size: 28px;
margin-top: 40px;
}
/* Paragraphs */
#about-us p {
color: #555; /* Medium gray for body text */
font-size: 18px;
line-height: 1.6;
margin-bottom: 20px;
text-align: justify;
}
/* Values List */
#about-us ul {
list-style: none;
padding: 0;
}
#about-us ul li {
background-color: #fff; /* White background for each value */
margin-bottom: 15px;
padding: 15px 20px;
border-left: 4px solid #2c3e50; /* Left border for emphasis */
box-shadow: 0 2px 10px rgba(0, 0, 0, 0.1);
font-size: 18px;
}
#about-us ul li strong {
color: #2c3e50; /* Strong emphasis for value title */
font-weight: bold;
}
/* Media Queries for Responsiveness */
@media (max-width: 768px) {
#about-us h2 {
font-size: 28px;
}
#about-us h3 {
font-size: 22px;
}
#about-us p {
font-size: 16px;
}
#about-us ul li {
font-size: 16px;
}
}
.card {
width: 350px;
border: 1px solid #ccc;
border-radius: 8px;
overflow: hidden;
box-shadow: 0 2px 8px rgba(0, 0, 0, 0.1);
background-color: #fff;
}
.card img {
width: 100%;
height: auto;
}
.card-body {
padding: 16px;
}
.card-title {
font-size: 18px;
font-weight: bold;
margin: 0;
}
.price {
font-size: 24px;
color: green;
margin: 8px 0;
}
.details {
font-size: 14px;
color: #555;
margin: 8px 0;
}
.visit-info {
font-size: 12px;
color: #777;
margin: 8px 0;
}
.provider {
display: flex;
align-items: center;
justify-content: space-between;
border-top: 1px solid #eee;
padding: 12px 16px;
}
.provider img {
width: 32px;
height: auto;
}
.provider .rating {
display: flex;
align-items: center;
}
.provider .rating span {
margin-left: 4px;
color: #555;
}
.book-button {
display: block;
width: calc(100% - 32px);
margin: 16px;
padding: 12px;
background-color: #ffbb33;
color: white;
text-align: center;
text-decoration: none;
font-weight: bold;
border-radius: 4px;
transition: background-color 0.3s;
}
.book-button:hover {
background-color: #ffbb33;
}

.reviews-newsletter {
background-color: #f8f9fa;
padding: 40px 20px;
border-top: 1px solid #ddd;
}

.container {
display: flex;
justify-content: space-between;
flex-wrap: wrap;
}

.reviews-section, .newsletter-section {
background-color: #ffffff;
padding: 20px;
border-radius: 8px;
box-shadow: 0 4px 6px rgba(0, 0, 0, 0.1);
width: 48%;
}

.reviews-section h2, .newsletter-section h2 {
font-size: 24px;
margin-bottom: 20px;
color: #333;
text-align: center;
border-bottom: 2px solid #ffbb33;
padding-bottom: 10px;
}

.review-form label, .newsletter-section label {
font-weight: bold;
color: #555;
margin-bottom: 8px;
display: block;
}

.review-form input, .review-form textarea, .review-form select,
.newsletter-section input {
width: 100%;
padding: 10px;
margin-bottom: 15px;
border: 1px solid #ddd;
border-radius: 4px;
box-sizing: border-box;
font-size: 16px;
}

.review-form button, .newsletter-section button {
background-color: #ffbb33;
color: #ffffff;
padding: 12px 20px;
border: none;
border-radius: 4px;
cursor: pointer;
font-size: 16px;
width: 100%;
}

.review-form button:hover, .newsletter-section button:hover {
background-color: #ffbb33;
}

.submitted-reviews {
margin-top: 20px;
}

.review-card {
background-color: #f1f1f1;
padding: 15px;
margin-bottom: 15px;
border-left: 4px solid #4CAF50;
border-radius: 8px;
}

.review-card h3 {
margin-top: 0;
font-size: 20px;
color: #333;
}

.review-card p {
margin: 10px 0;
color: #555;
}

.review-card .rating {
color: #ffbb33;
font-size: 18px;
}

.newsletter-section p {
font-size: 16px;
color: #555;
margin-bottom: 15px;
text-align: center;
}

.newsletter-section form {
text-align: center;
}

.newsletter-section input[type="email"] {
max-width: 300px;
margin-bottom: 15px;
}

/* Circular Blog Section */
.circular-blog-section {
display: flex;
flex-direction: column;
align-items: center;
justify-content: center;
height: 100vh;
background-color: #f4f4f9;
text-align: center;
padding: 20px;
}

.section-heading {
font-size: 4rem;
color: var(--orange);
margin-bottom: 2rem;
}

.circular-blog-card {
width: 450px; /* Increased size */
height: 450px; /* Increased size */
border-radius: 50%;
overflow: hidden;
box-shadow: 0 10px 20px rgba(0, 0, 0, 0.1);
position: relative;
display: flex;
justify-content: center;
align-items: center;
background-color: white;
transition: transform 0.5s ease;
}

.blog-content {
text-align: center;
padding: 20px;
opacity: 0;
transition: opacity 0.5s ease;
position: absolute;
color: #333;
width: 90%;
}

.blog-image {
width: 80%;
height: 70%;
border-radius: 8px;
margin-bottom: 1rem;
}

.blog-title {
font-size: 1.8rem;
color: var(--orange);
margin-bottom: 1rem;
}

.blog-excerpt {
font-size: 1.5rem;
color: #080808;
line-height: 1.6;
}

.blog-content.active {
opacity: 1;
z-index: 1;
}

.circular-blog-card:hover {
transform: rotate(360deg);
}

.modal {
display: none;
position: fixed;
z-index: 1;
left: 0;
top: 0;
width: 100%;
height: 100%;
overflow: auto;
background-color: rgba(0, 0, 0, 0.4);
}

.modal-content {
background-color: #fefefe;
margin: 15% auto;
padding: 20px;
border: 1px solid #888;
width: 80%;
max-width: 600px;
}

.close {
color: #aaa;
float: right;
font-size: 28px;
font-weight: bold;
}

.close:hover,
.close:focus {
color: black;
text-decoration: none;
cursor: pointer;
}

/* Payment Section */
.payment-section {
text-align: center;
padding: 20px;
background-color: #f9f9f9;
}

.payment-section h2 {
font-size: 24px;
margin-bottom: 20px;
color: #333;
}

/* Payment Container */
.payment-container {
display: flex;
justify-content: center;
align-items: center;
gap: 20px;
flex-wrap: wrap;
margin-top: 20px;
}

/* Payment Button Styles */
.payment-container button {
background-color: #ffffff;
border: 1px solid #ddd;
border-radius: 10px;
padding: 10px;
box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
cursor: pointer;
transition: transform 0.2s, box-shadow 0.2s;
width: 80px;
height: 80px;
display: flex;
justify-content: center;
align-items: center;
}

.payment-container button:hover {
transform: scale(1.1);
box-shadow: 0 6px 12px rgba(0, 0, 0, 0.15);
}

/* Logo Styles */
.payment-container img {
max-width: 100%;
max-height: 100%;
display: block;
object-fit: contain;
}



On Sat, Sep 7, 2024 at 4:17 PM Aballo Janet ***@***.***>
wrote:

> https://meet.google.com/nzv-srrb-yue
>
> On Sat, Sep 7, 2024 at 12:48 PM Abili Isaac ***@***.***>
> wrote:
>
> > @Abili <https://github.com/Abili> requested your review on: #4
> > <https://github.com/happy-janet/Raha-Voyage-Safaris/pull/4> Updates.
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <
> https://github.com/happy-janet/Raha-Voyage-Safaris/pull/4#event-14168988379>,
>
> > or unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/BGJTZLZ4BFZUF2Z2AOCLUM3ZVLDVBAVCNFSM6AAAAABNZ36NM6VHI2DSMVQWIX3LMV45UABCJFZXG5LFIV3GK3TUJZXXI2LGNFRWC5DJN5XDWMJUGE3DQOJYHAZTOOI>
>
> > .
> > You are receiving this because your review was requested.Message ID:
> > ***@***.***
> > com>
> >
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/happy-janet/Raha-Voyage-Safaris/pull/4#issuecomment-2335184408>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADKB7W4JM5TCNKZRAMCXOFLZVL4HXAVCNFSM6AAAAABNZ36NM6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDGMZVGE4DINBQHA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

Abili commented:
https://github.com/aksalj/pesapaljs

check out this pesapal implementation see if u can get some pointers from it

On Sat, Sep 7, 2024 at 4:53 PM Abili Isaac ***@***.***> wrote:

> @import url('
> ***@***.***;300;400;600;700&display=swap
> ');
>
> :root{
> --orange:#ffa500;
> }
>
>
> *{
> font-family: 'Nunito', sans-serif;
> margin: 0px;
> padding: 0px;
> box-sizing: border-box;
> text-transform: capitalize;
> outline: none;;
> border: none;
> text-decoration: none;
> transition:all .2s linear;
> }
> *::selection{
> background: var(--orange);
> color: white;
> }
> html{
> font-size: 62.5%;
> overflow-x: hidden;
> scroll-padding-top: 6rem;
> scroll-behavior: smooth;
> }
> section{
> padding:2rem 9%;
>
> }
> .heading{
> text-align: center;
> padding: 2.5rem 0;
> }
> .heading span{
> font-size: 3.5rem;
> background: rgba(255, 165, 0, .2);
> color: var(--orange);
> border-radius: .5rem;
> padding: .2rem 1rem;
> }
> .heading span.space{
> background: none;
> }
> /* Header styling */
> header {
> display: flex;
> align-items: center;
> padding: 20px;
> background-color: #2c3e50; /* Dark background for the header */
> }
> #menu-bar {
> font-size: 24px;
> color: #fff; /* White color for the menu icon */
> margin-right: 20px;
> cursor: pointer;
> text-align: left;
> }
> .logo {
> display: flex;
> align-items: center;
> text-decoration: none;
> font-size: 24px;
> color: #fff;
> font-family: 'Arial', sans-serif;
> }
> .logo-image {
> width: 50px; /* Adjust size according to your logo */
> height: auto;
> margin-right: 10px; /* Space between logo and text */
> }
> .logo span {
> font-weight: bold;
> color: #e74c3c; /* Color for 'Raha Voyage' */
> }
> .logo {
> color: #fff; /* Color for 'Safaris' */
> }
> /* Media Query for smaller screens */
> @media (max-width: 768px) {
> header {
> flex-direction: column;
> align-items: flex-start;
> }
> .logo-image {
> width: 40px; /* Smaller size for the logo on mobile */
> }
> .logo {
> font-size: 20px;
> }
> #menu-bar {
> margin-bottom: 10px;
> }
> }
> header{
> position: fixed;
> top: 0; left: 0; right: 0;
> background: #333;
> z-index: 1000;
> display: flex;
> align-items: center;
> justify-content: space-between;
> padding: 2rem 9%;
> }
> header .logo{
> font-size: 2.5rem;
> font-weight: bolder;
> color: white;
> text-transform: uppercase;
> }
>
> header .logo span{
> color: var(--orange);
> }
> header .navbar a{
> color: white;
> font-size: 2rem;
> margin: 0 .8rem;
> }
> header .navbar a:hover{
> color: var(--orange);
> }
> header.icons i{
> font-size: 2.5rem;
> color: white;
> cursor: pointer;
> margin-right: 2rem;
> }
> header .icons i:hover{
> color: var(--orange);
> }
> header .search-bar-container{
> position: absolute;
> top: 100%; left: 0; right: 0;
> padding: 1.5rem 2rem;
> background: #333;
> border-top: .1rem solid rgba( 255,255,255,.2);
> display: flex;
> align-items: center;
> z-index: 1001;
> clip-path: polygon(0 0,100% 0, 100% 0, 0 0);
> }
> header .search-bar-container.active{
> clip-path: polygon(0 0,100% 0, 100% 100%, 0 100%);
> }
> header .search-bar-container #search-bar{
> width: 100%;
> padding: 1rem;
> text-transform: none;
> color: #333;
> font-size: 1.7rem;
> border-radius: .5rem;
> }
> header .search-bar-container label{
> color: white;
> cursor: pointer;
> font-size: 3rem;
> margin-left: 1.5rem;
> }
> header .search-bar-container label:hover{
> color: var(--orange);
> }
> .btn{
> display: inline-block;
> margin-top: 1rem;
> background: var(--orange);
> color: white;
> padding: .8rem 3rem;
> border: .2rem solid var(--orange);
> cursor: pointer;
> font-size: 1.7rem;
> }
> .btn:hover{
> background: rgba(255, 165, 0, .2);
> color:var(--orange);
> }
> .login-form-container{
> position:fixed;
> top: -120%; left: 0;
> z-index: 10000;
> min-height: 100vh;
> width: 100%;
> background: rgba(0,0,0,0.7);
> display:flex;
> align-items: center;
> justify-content: center;
> }
> .login-form-container.active{
> top: 0;
> }
> .login-form-container form{
> margin:2rem;
> padding: 1.5rem 2rem;
> border-radius: .5rem;
> background: white;
> width: 50rem;
> }
>
> .login-form-container form h3{
> font-size: 3rem;
> color: #444;
> text-transform:uppercase;
> text-align: center;
> padding: 1rem 0;
> }
> .login-form-container form .box{
> width:100%;
> padding: 1rem;
> font-size: 1.7rem;
> color: #333;
> margin: .6rem 0;
> border: .1rem solid rgba(0,0,0,0.3);
> text-decoration: none;
> }
> .login-form-container form .box:focus{
> border-color: var(--orange);
> }
> .login-form-container form #remember{
> margin: 2rem 0;
> }
> .login-form-container form label{
> font-size: 1.5rem;
> }
> .login-form-container form .btn{
> display: block;
> width: 100%;
> }
> .login-form-container form p{
> padding: .5rem 0;
> font-size: 1.5rem;
> color: #666;
> }
> .login-form-container form p a{
> color: var(--orange);
> }
> .login-form-container form p a:hover{
> color: #333;
> text-decoration: underline;
> }
> .login-form-container #form-close{
> position: absolute;
> top:2rem; right: 3rem;
> font-size: 5rem;
> color:white;
> cursor: pointer;
> }
> #menu-bar{
> color: white;
> border: .1rem solid white;
> border-radius: .5rem;
> font-size: 3rem;
> padding: .5rem 1.2rem;
> cursor: pointer;
> display: none;
> }
>
> .home{
> min-height: 100vh;
> display: flex;
> align-items: center;
> justify-content: center;
> flex-flow: column;
> position: relative;
> z-index: 0;
> }
> .home .content{
> text-align: center;
> }
> .home .content h3{
> font-size: 4.5rem;
> color: white;
> text-transform: uppercase;
> text-shadow: 0 .3rem .5rem rgba(0,0,0,0.1);
> }
> .home .content p{
> font-size: 2.5rem;
> color: white;
> padding: 5rem 0;
> }
> .home .video-container video{
> position: absolute;
> top: 0; left: 0;
> z-index: -1;
> height: 100%;
> width: 100%;
> object-fit: cover;
> }
>
> .home .controls{
> padding: 1rem;
> border-radius: 5rem;
> background: rgba(0,0,0,0.7);
> position: relative;
> top: 10rem;
> }
> .home .controls .vid-btn{
> height: 2rem;
> width: 2rem;
> display: inline-block;
> border-radius: 50%;
> background: white;
> cursor: pointer;
> margin: 0 .5rem;
> }
>
> .home .controls .vid-btn.active{
> background: var(--orange);
> }
>
> .book .row{
> display: flex;
> flex-wrap: wrap;
> gap:1.5rem;
> align-items: center;
> }
>
> .book .row .image{
> flex: 1 1 40rem;
> }
> .book .row .image img{
> width: 100%;
> height: 550px;
> }
> .book .row form{
> flex: 1 1 40rem;
> padding: 2rem;
> box-shadow: 0 1rem 2rem rgba(0,0,0,0.1);
> border-radius: .5rem;
> }
> .book .row form .inputBox{
> padding: .5rem 0;
> }
> .book .row form .inputBox input{
> width: 100%;
> padding: 1rem;
> border: 1rem solid rgba(0,0,0,0.1);
> font-size: 1rem;
> color: #333;
> text-transform: none;
> }
> .book .row form .inputBox h3{
> font-size: 2rem;
> padding: 1rem 0;
> color: #666;
> }
>
> .packages .box-container{
> display: flex;
> flex-wrap: wrap;
> gap: 2rem;
> }
> .packages .box-container .box{
> flex: 1 1 30rem;
> border-radius: .5rem;
> overflow: hidden;
> box-shadow: 0 1rem 2rem rgba(0,0,0,0.1);
> }
> .packages .box-container .box img{
> height:25rem;
> width: 100%;
> object-fit: cover;
> }
> .packages .box-container .box .content{
> padding: 2rem;
> }
> .packages .box-container .box .content h3{
> font-size: 2rem;
> color: #333;
> }
> .packages .box-container .box .content h3 i{
> color: var(--orange);
> }
>
> .packages .box-container .box .content p{
> font-size: 1.7rem;
> color: #333;
> padding: 1rem 0;
> }
> .packages .box-container .box .content .stars i{
> font-size: 1.7rem;
> color: var(--orange);
> }
> .packages .box-container .box .content .price{
> font-size: 2rem;
> color: #333;
> padding-top: 1rem;
> }
>
> .packages .box-container .box .content .price span{
> color: #666;
> font-size: 1.5rem;
> text-decoration: line-through;
> }
>
> .services .box-container{
> display: flex;
> flex-wrap: wrap;
> gap:1.5rem;
> }
> .services .box-container .box{
> flex: 1 1 30rem;
> border-radius: .5rem;
> padding: 1rem;
> text-align: center;
>
> }
>
> .services .box-container .box i{
> padding:1rem;
> font-size: 5rem;
> color: var(--orange);
> }
> .services .box-container .box h3{
> font-size: 2.5rem;
> color: #333;
> }
>
> .services .box-container .box p{
> padding: 1rem 0;
> font-size: 1.5rem;
> color: #666;
> }
>
> .services .box-container .box:hover{
> box-shadow: 0 1rem 2rem rgba(0,0,0,0.1);
> }
>
> .gallery .box-container{
> display: flex;
> flex-wrap: wrap;
> gap: 1.5rem;
> }
> .gallery .box-container .box{
> overflow: hidden;
> box-shadow: 0 1rem 2rem rgba(0,0,0,0.1);
> border: 1rem solid white;
> border-radius: .5rem;
> flex: 1 1 30rem;
> height: 25rem;
> position:relative;
> }
>
> .gallery .box-container .box img{
> height: 100%;
> width: 100%;
> object-fit: cover;
> }
> .gallery .box-container .box .content{
> position: absolute;
> top: -100%; left: 0;
> height: 100%;
> width: 100%;
> text-align: center;
> background: rgba(0,0,0,0.7);
> padding: 2rem;
> padding-top: 5rem;
> }
> .gallery .box-container .box:hover .content{
> top: 0;
> }
> .gallery .box-container .box .content h3{
> font-size: 2.5rem;
> color: var(--orange);
> }
> .gallery .box-container .box .content p{
> font-size: 1.5rem;
> color: white;
> padding: .5rem 0;
> }
>
> .review .box{
> padding: 2rem;
> text-align: center;
> box-shadow: 0 1rem 2rem rgba(0,0,0,0.1);
> border-radius: .5rem;
> }
>
> .review .box img{
> height: 13rem;
> width: 13rem;
> border-radius: 50%;
> object-fit: cover;
> margin-bottom: 1rem;
> }
>
> .review .box h3{
> color: #ffa500;
> font-size: 2.5rem;
> }
> .review .box p{
> color: #666;
> font-size: 1.5rem;
> padding: 1rem 0;
> }
> .review .box .stars i{
> color: var(--orange);
> font-size: 1.7rem;
> }
>
> .contact .row{
> display: flex;
> flex-wrap: wrap;
> gap: 1.5rem;
> align-items: center;
> }
> .contact .row .image{
> flex: 1 1 35rem;
> }
> .contact .row .image img{
> width: 100%;
> height: 350px;
> }
> .contact .row form{
> flex: 1 1 50rem;
> padding: 2rem;
> box-shadow: 0 1rem 2rem rgba(0,0,0,0.1);
> border-radius: .5rem;
> }
>
> .contact .row form .inputBox{
> display: flex;
> flex-wrap: wrap;
> justify-content: space-between;
> }
> .contact .row form ,.inputBox input, .contact .row form textarea{
> width: 49%;
> margin: 1rem 0;
> padding: 1rem;
> font-size: 1.7rem;
> color: #333;
> background: #f7f7f7;
> text-transform: none;
> }
>
> .contact .row form textarea{
> height: 15rem;
> resize: none;
> width: 100%;
> }
>
>
>
> .footer{
> background: #333;
> }
>
> .footer .box-container{
> display: flex;
> flex-wrap: wrap;
> gap: 1.5rem;
> }
> .footer .box-container .box{
> padding: 1rem 0;
> flex: 1 1 25rem;
> }
> .footer .box-container .box h3{
> font-size: 2.5rem;
> padding: .7rem 0;
> color: white;
> }
> .footer .box-container .box p{
> font-size: 1.5 rem;
> padding: .7rem 0;
> color: white;
> }
> .footer .box-container .box a{
> display: block;
> font-size: 1.5rem;
> padding: .7rem 0;
> color: white;
> }
> .footer.box-container .box a:hover{
> color: var(--orange);
> text-decoration: underline;
> }
>
> .footer .credit{
> text-align: center;
> padding: 2rem 1rem;
> margin-top: 1rem;
> font-size: 2rem;
> font-weight: normal;
> color: white;
> border-top: .1rem solid rgba( 255,255,255,.2)
> }
> .footer .credit span{
> color: var(--orange);
> }
>
>
>
>
>
>
> /* media queries */
>
> @media (max-width:1200px){
> html{
> font-size: 55%;
> }
> }
> @media (max-width:991px){
> header{
> padding: 2rem;
> }
> section{
> padding: 2rem;
> }
> }
> @media (max-width:768px){
> #menu-bar{
> display: initial;
> }
> header .navbar{
> position: absolute;
> top: 100%; right: 0; left: 0;
> background: #333;
> border-top: .1rem solid rgba( 255,255,255,.2);
> padding: 1rem 2rem;
> clip-path:polygon(0 0, 100% 0, 100% 0,0 0) ;
> }
> header .navbar.active{
> clip-path: polygon(0 0,100% 0, 100% 100%, 0 100%);
> }
> header .navbar a{
> display: block;
> border-radius: .5rem;
> padding: 1.5rem;
> margin: 1.5rem 0;
> background: #222;
> }
> }
> @media (max-width:450px){
> html{
> font-size: 50%;
> }
>
>
> .heading span{
> font-size: 2.5rem;
> }
> .contact .row form .inputBox input{
> width: 100%;
> }
> }
>
>
> /* General container styling */
> #about-us {
> padding: 50px 20px;
> background-color: #f9f9f9; /* Light background color */
> font-family: 'Arial', sans-serif;
> }
> #about-us .container {
> max-width: 1200px;
> margin: 0 auto;
> }
> /* Headings */
> #about-us h2, #about-us h3 {
> color: #ffa500; /* Darker color for headings */
> text-align: center;
> margin-bottom: 20px;
> }
> #about-us h2 {
> font-size: 36px;
> }
> #about-us h3 {
> font-size: 28px;
> margin-top: 40px;
> }
> /* Paragraphs */
> #about-us p {
> color: #555; /* Medium gray for body text */
> font-size: 18px;
> line-height: 1.6;
> margin-bottom: 20px;
> text-align: justify;
> }
> /* Values List */
> #about-us ul {
> list-style: none;
> padding: 0;
> }
> #about-us ul li {
> background-color: #fff; /* White background for each value */
> margin-bottom: 15px;
> padding: 15px 20px;
> border-left: 4px solid #2c3e50; /* Left border for emphasis */
> box-shadow: 0 2px 10px rgba(0, 0, 0, 0.1);
> font-size: 18px;
> }
> #about-us ul li strong {
> color: #2c3e50; /* Strong emphasis for value title */
> font-weight: bold;
> }
> /* Media Queries for Responsiveness */
> @media (max-width: 768px) {
> #about-us h2 {
> font-size: 28px;
> }
> #about-us h3 {
> font-size: 22px;
> }
> #about-us p {
> font-size: 16px;
> }
> #about-us ul li {
> font-size: 16px;
> }
> }
> .card {
> width: 350px;
> border: 1px solid #ccc;
> border-radius: 8px;
> overflow: hidden;
> box-shadow: 0 2px 8px rgba(0, 0, 0, 0.1);
> background-color: #fff;
> }
> .card img {
> width: 100%;
> height: auto;
> }
> .card-body {
> padding: 16px;
> }
> .card-title {
> font-size: 18px;
> font-weight: bold;
> margin: 0;
> }
> .price {
> font-size: 24px;
> color: green;
> margin: 8px 0;
> }
> .details {
> font-size: 14px;
> color: #555;
> margin: 8px 0;
> }
> .visit-info {
> font-size: 12px;
> color: #777;
> margin: 8px 0;
> }
> .provider {
> display: flex;
> align-items: center;
> justify-content: space-between;
> border-top: 1px solid #eee;
> padding: 12px 16px;
> }
> .provider img {
> width: 32px;
> height: auto;
> }
> .provider .rating {
> display: flex;
> align-items: center;
> }
> .provider .rating span {
> margin-left: 4px;
> color: #555;
> }
> .book-button {
> display: block;
> width: calc(100% - 32px);
> margin: 16px;
> padding: 12px;
> background-color: #ffbb33;
> color: white;
> text-align: center;
> text-decoration: none;
> font-weight: bold;
> border-radius: 4px;
> transition: background-color 0.3s;
> }
> .book-button:hover {
> background-color: #ffbb33;
> }
>
> .reviews-newsletter {
> background-color: #f8f9fa;
> padding: 40px 20px;
> border-top: 1px solid #ddd;
> }
>
> .container {
> display: flex;
> justify-content: space-between;
> flex-wrap: wrap;
> }
>
> .reviews-section, .newsletter-section {
> background-color: #ffffff;
> padding: 20px;
> border-radius: 8px;
> box-shadow: 0 4px 6px rgba(0, 0, 0, 0.1);
> width: 48%;
> }
>
> .reviews-section h2, .newsletter-section h2 {
> font-size: 24px;
> margin-bottom: 20px;
> color: #333;
> text-align: center;
> border-bottom: 2px solid #ffbb33;
> padding-bottom: 10px;
> }
>
> .review-form label, .newsletter-section label {
> font-weight: bold;
> color: #555;
> margin-bottom: 8px;
> display: block;
> }
>
> .review-form input, .review-form textarea, .review-form select,
> .newsletter-section input {
> width: 100%;
> padding: 10px;
> margin-bottom: 15px;
> border: 1px solid #ddd;
> border-radius: 4px;
> box-sizing: border-box;
> font-size: 16px;
> }
>
> .review-form button, .newsletter-section button {
> background-color: #ffbb33;
> color: #ffffff;
> padding: 12px 20px;
> border: none;
> border-radius: 4px;
> cursor: pointer;
> font-size: 16px;
> width: 100%;
> }
>
> .review-form button:hover, .newsletter-section button:hover {
> background-color: #ffbb33;
> }
>
> .submitted-reviews {
> margin-top: 20px;
> }
>
> .review-card {
> background-color: #f1f1f1;
> padding: 15px;
> margin-bottom: 15px;
> border-left: 4px solid #4CAF50;
> border-radius: 8px;
> }
>
> .review-card h3 {
> margin-top: 0;
> font-size: 20px;
> color: #333;
> }
>
> .review-card p {
> margin: 10px 0;
> color: #555;
> }
>
> .review-card .rating {
> color: #ffbb33;
> font-size: 18px;
> }
>
> .newsletter-section p {
> font-size: 16px;
> color: #555;
> margin-bottom: 15px;
> text-align: center;
> }
>
> .newsletter-section form {
> text-align: center;
> }
>
> .newsletter-section input[type="email"] {
> max-width: 300px;
> margin-bottom: 15px;
> }
>
> /* Circular Blog Section */
> .circular-blog-section {
> display: flex;
> flex-direction: column;
> align-items: center;
> justify-content: center;
> height: 100vh;
> background-color: #f4f4f9;
> text-align: center;
> padding: 20px;
> }
>
> .section-heading {
> font-size: 4rem;
> color: var(--orange);
> margin-bottom: 2rem;
> }
>
> .circular-blog-card {
> width: 450px; /* Increased size */
> height: 450px; /* Increased size */
> border-radius: 50%;
> overflow: hidden;
> box-shadow: 0 10px 20px rgba(0, 0, 0, 0.1);
> position: relative;
> display: flex;
> justify-content: center;
> align-items: center;
> background-color: white;
> transition: transform 0.5s ease;
> }
>
> .blog-content {
> text-align: center;
> padding: 20px;
> opacity: 0;
> transition: opacity 0.5s ease;
> position: absolute;
> color: #333;
> width: 90%;
> }
>
> .blog-image {
> width: 80%;
> height: 70%;
> border-radius: 8px;
> margin-bottom: 1rem;
> }
>
> .blog-title {
> font-size: 1.8rem;
> color: var(--orange);
> margin-bottom: 1rem;
> }
>
> .blog-excerpt {
> font-size: 1.5rem;
> color: #080808;
> line-height: 1.6;
> }
>
> .blog-content.active {
> opacity: 1;
> z-index: 1;
> }
>
> .circular-blog-card:hover {
> transform: rotate(360deg);
> }
>
> .modal {
> display: none;
> position: fixed;
> z-index: 1;
> left: 0;
> top: 0;
> width: 100%;
> height: 100%;
> overflow: auto;
> background-color: rgba(0, 0, 0, 0.4);
> }
>
> .modal-content {
> background-color: #fefefe;
> margin: 15% auto;
> padding: 20px;
> border: 1px solid #888;
> width: 80%;
> max-width: 600px;
> }
>
> .close {
> color: #aaa;
> float: right;
> font-size: 28px;
> font-weight: bold;
> }
>
> .close:hover,
> .close:focus {
> color: black;
> text-decoration: none;
> cursor: pointer;
> }
>
> /* Payment Section */
> .payment-section {
> text-align: center;
> padding: 20px;
> background-color: #f9f9f9;
> }
>
> .payment-section h2 {
> font-size: 24px;
> margin-bottom: 20px;
> color: #333;
> }
>
> /* Payment Container */
> .payment-container {
> display: flex;
> justify-content: center;
> align-items: center;
> gap: 20px;
> flex-wrap: wrap;
> margin-top: 20px;
> }
>
> /* Payment Button Styles */
> .payment-container button {
> background-color: #ffffff;
> border: 1px solid #ddd;
> border-radius: 10px;
> padding: 10px;
> box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
> cursor: pointer;
> transition: transform 0.2s, box-shadow 0.2s;
> width: 80px;
> height: 80px;
> display: flex;
> justify-content: center;
> align-items: center;
> }
>
> .payment-container button:hover {
> transform: scale(1.1);
> box-shadow: 0 6px 12px rgba(0, 0, 0, 0.15);
> }
>
> /* Logo Styles */
> .payment-container img {
> max-width: 100%;
> max-height: 100%;
> display: block;
> object-fit: contain;
> }
>
>
>
> On Sat, Sep 7, 2024 at 4:17 PM Aballo Janet ***@***.***>
> wrote:
>
>> https://meet.google.com/nzv-srrb-yue
>>
>> On Sat, Sep 7, 2024 at 12:48 PM Abili Isaac ***@***.***>
>> wrote:
>>
>> > @Abili <https://github.com/Abili> requested your review on: #4
>> > <https://github.com/happy-janet/Raha-Voyage-Safaris/pull/4> Updates.
>> >
>> > —
>> > Reply to this email directly, view it on GitHub
>> > <
>> https://github.com/happy-janet/Raha-Voyage-Safaris/pull/4#event-14168988379>,
>>
>> > or unsubscribe
>> > <
>> https://github.com/notifications/unsubscribe-auth/BGJTZLZ4BFZUF2Z2AOCLUM3ZVLDVBAVCNFSM6AAAAABNZ36NM6VHI2DSMVQWIX3LMV45UABCJFZXG5LFIV3GK3TUJZXXI2LGNFRWC5DJN5XDWMJUGE3DQOJYHAZTOOI>
>>
>> > .
>> > You are receiving this because your review was requested.Message ID:
>> > ***@***.***
>> > com>
>> >
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/happy-janet/Raha-Voyage-Safaris/pull/4#issuecomment-2335184408>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/ADKB7W4JM5TCNKZRAMCXOFLZVL4HXAVCNFSM6AAAAABNZ36NM6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDGMZVGE4DINBQHA>
>> .
>> You are receiving this because you were mentioned.Message ID:
>> ***@***.***>
>>
>
